bot/grenouille_irc_bot.py

```diff
@@ -79,6 +81,9 @@ def on_ping(self, connection, e):
         """
         self.last_ping = datetime.utcnow()
 
+    def send_msg(self, line):
+        self.connection.privmsg(self.channel, line)
```

Comment:
We will have to secure the `send_msg` somehow. If we send the message when the bot is offline (sometimes), the process will crash. That's just a sidenote for you, I will do a small patch to improve downtime behavior.